charts/newrelic-logging/templates/daemonset.yaml

```diff
@@ -44,6 +44,10 @@ spec:
       securityContext:
         {{- . | nindent 8 }}
       {{- end }}
+      {{- if .Values.initContainers }}
+      initContainers:
+{{ toYaml .Values.initContainers | indent 8 }}
+      {{- end }}
```

Comment:
Please also add this to the Windows daemonset (`daemonset-windows.yml`) as well. Please bear in mind that that file initializes an _array_ of Daemonsets, so you'll need to use `$.Values.initContainers` instead.